samples/subsys/usb/mass/prj.conf

```diff
@@ -1,6 +1,6 @@
 CONFIG_STDOUT_CONSOLE=y
 CONFIG_SYS_LOG=y
-CONFIG_SYS_LOG_DEFAULT_LEVEL=1
+CONFIG_SYS_LOG_DEFAULT_LEVEL=3
```

Comment:
Please leave SYS_LOG_LEVEL as it was.

Reply:
@ajaykish what does 'internal test automation' mean?  Why doesn't your internal test automation set this to the level it wants?

Reply:
> @jfischer-phytec-iot This is required for internal test automation.

This is not a good enough reason, this is a sample, not a test and making changes here does affect everyone using it. Depending on a certain log level in a sample might not provide a solid  test environment.